misc/teleop_twist_keyboard/launch/keyboard_teleop.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+        <!-- Keyboard teleop launch -->
+	<node name= "teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" ns="caffeine" output="screen"/>
```

Comment:
I believe this will still publish to `cmd_vel` and not `caffeine/cmd_vel`. Have you tested it?

Reply:
as we discussed, fix indentation and use an `arg` for the node's `ns`